mmv1/third_party/terraform/website/docs/d/composer_user_workloads_secret.html.markdown

```diff
@@ -0,0 +1,60 @@
+---
+subcategory: "Cloud Composer"
+description: |-
+  Kubernetes secret in Composer Environment workloads.
```

Comment:
Please make "secret" capfirst as "Secret" here and in other places of this file

Reply:
fixed